core-blocks/test/fixtures/core__latest-comments.serialized.html

```diff
@@ -0,0 +1 @@
+<!-- wp:latest-comments /-->
```

Comment:
I think we need to regenerate block definition for tests. You can do it with the following command:
`npm run fixtures:regenerate`